castle/cms/browser/controlpanel/templates/status.pt

```diff
@@ -0,0 +1,59 @@
+
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      lang="en"
+      metal:use-macro="context/prefs_main_template/macros/master"
+      i18n:domain="plone">
+<body>
+<metal:main metal:fill-slot="prefs_configlet_main">
+
+    <a href=""
+       id="setup-link"
+       tal:attributes="href string:$portal_url/plone_control_panel"
+       i18n:translate="">
+        Site Setup
+    </a>
+
+
+
+    <h1> Subprocesses </h1>
+
+    <div metal:use-macro="context/global_statusmessage/macros/portal_message">
+      Portal status message
+    </div>
+
+    <div id="content-core">
+        <div tal:condition="python:view.docsplit()">
+            <span class="glyphicon glyphicon-ok" aria-hidden="true"></span> Docsplit: OK
+        </div>
+    </div>
+    <div id="content-core">
```

Comment:
The `id` attribute of html elements should be unique within the document. This explains why this might matter https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id . You should be able to get away with just having one `<div id="content-core">` to wrap all of the <div> that contain tal:condition.